website/announcements/models.py

```diff
@@ -188,9 +189,12 @@ class Slide(models.Model):
         verbose_name=_("Display only for authenticated members"), default=False
     )
 
-    url = models.URLField(
+    custom_url = models.URLField(
```

Comment:
I've got a hunch that this needs to be changed in the slide admin as well?

Reply:
It seems the field is not listed there, SlideAdmin uses an implicit list of fields